service-automation/src/com/appyschool/tests/students/TestStudent.java

```diff
@@ -1,46 +1,225 @@
 package com.appyschool.tests.students;
 
 import java.util.HashMap;
-
+import java.util.Map;
 
 import org.apache.commons.lang3.RandomStringUtils;
+import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 import org.testng.annotations.Test;
 
+import com.appyschool.common.CommonMethods;
 import com.appyschool.common.CommonUtils;
+import com.appyschool.common.ConfigProperties;
 import com.appyschool.common.DataProviderArguments;
+import com.appyschool.common.GenericConstants;
+import com.appyschool.common.SoftAssertionBase;
+import com.appyschool.common.SuiteSetupProperties;
+import com.appyschool.services.AdhocServices;
 import com.appyschool.services.BranchServices;
+import com.appyschool.services.SectionServices;
+import com.appyschool.services.StandardServices;
 import com.appyschool.services.StudentServices;
+import com.appyschool.services.SubjectServices;
 import com.jayway.restassured.response.Response;
 
 public class TestStudent {
 
+
```

Comment:
not sure what you have written here? looks like copy-pasted from testsuitesetup java file

Reply:
Initially I was too confused abut "dependson" and other things ,so I asked Vikash jiju about this and he said that TestStudent was just an example to make things clear for me and main task is of  TestStaff and I can use TestSuite in TestStudent so that I get better picture of everything happeing.